tests/filters/test_proportion.py

```diff
@@ -10,17 +10,24 @@
 
 
 from preggy import expect
+from tornado.testing import gen_test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'tornado.testing'](https://app.codacy.com/manual/heynemann/thumbor/pullRequest?prid=4967909)